lib/SQL/Translator/Parser/MySQL.pm

```diff
@@ -326,7 +326,7 @@ create : CREATE PROCEDURE NAME not_delimiter "$delimiter"
 PROCEDURE : /procedure/i
     | /function/i
 
-create : CREATE or_replace(?) create_view_option(s?) /view/i NAME /as/i view_select_statement "$delimiter"
+create : CREATE or_replace(?) create_view_option(s?) /view/i NAME parens_field_list(?) /as/i view_select_statement "$delimiter"
```

Comment:
An explicit field list overrides the names derived from the `SELECT` column list, so the field list needs to be saved here and stored in the `fields` attribute of the `SQL::Translator::Schema::View` object.